### DISCUSSION TITLE
Bios for RG350M

### DISCUSSION THREAD

**Discussion by tony28** [eduardofilo/RG350_adam_image#236] [General]
Hello, 
I have install Adam image but i don't know where to find the bios. 
Could you give an help ? 
Thank you
Tony

**Comment by neilswann80** [comment]
https://letmegooglethat.com/?q=Latest+Retroarch+bios+pack

;-)

**Comment by tony28** [comment]
Gracias :)

**Reply by neilswann80** [reply]
No Problem!  

FYI... your last post included some email metadata with your name and mobile number. Just in case you're concerned with privacy you may want to edit and remove that personal data.  If that is your legit mobile number it's pretty awesome too!

All the best and happy gaming!

**Comment by tony28** [comment]
Thank you, you're right :) I have answered with my email :)
